build.sbt

```diff
@@ -49,7 +49,7 @@ lazy val root = (project in file("."))
       "org.playframework"            %% "play-ws-standalone"      % playWsVersion,
       "org.playframework"            %% "play-ws-standalone-json" % playWsVersion,
       "com.typesafe.scala-logging"   %% "scala-logging"           % "3.9.5",
-      "io.lemonlabs"                 %% "scala-uri"               % "4.0.3",
+      "com.indoorvivants"                 %% "scala-uri"               % "4.2.0",
```

Comment:
Omg @ramazanyich . I didn't know the scala steward can detect groupId changes. I don't find anything spectacular under https://github.com/indoorvivants/scala-uri/releases so i guess we can merge this PR